src/components/Navbar/NavBar.tsx

```diff
@@ -296,10 +318,65 @@ const ReactNavLink = styled(Link)`
     color: black;
   }
   :hover {
-    color: #9e9e9e;
-  }
+    color: #9e9e9e;}
+  position:relative;
 `;
 
+
 const Close = styled(CloseIcon)`
   cursor: pointer;
+`;
+
+
+const Dropdown = styled.div`
+  display:flex;
+  z-index: 11;
```

Comment:
What's the z index for?

Reply:
Without the z index, the DescriptionBox would be on top of the dropdown container on the merchant page.

Reply:
I see. Mind making a constant called $max-z-index or something and making it 9999 or something, and importing that?